client/app/components/groups/ListItemAddon.jsx

```diff
@@ -4,16 +4,32 @@ import Tooltip from "antd/lib/tooltip";
 
 export default function ListItemAddon({ isSelected, isStaged, alreadyInGroup, deselectedIcon }) {
   if (isStaged) {
-    return <i className="fa fa-remove" />;
+    return (
+      <>
+        <i className="fa fa-remove" aria-hidden="true" />
+        <span className="sr-only">Remove</span>
+      </>
+    );
   }
   if (alreadyInGroup) {
     return (
       <Tooltip title="Already selected">
-        <i className="fa fa-check" />
+        <i className="fa fa-check" aria-hidden="true" />
+        <span className="sr-only">Already selected</span>
```

Comment:
I feel like tooltips should have accessibility support by Antd rather than us adding manually to each of our uses. However it doesn't 😕, so I'm no so sure how to proceed anyway. Maybe we should have a component to make it easier to drop off once Antd starts to have accessible tooltips?

Reply:
Tooltips are sort of a pain I will try to handle separately. In the meantime I'm uncertain whether I should provide a minimal approach to them or wait until they can be dealt with. Probably I'll just remove these changes.